utils/luci-app-owm/luasrc/owm.lua

```diff
@@ -205,6 +205,15 @@ function showmac(mac)
 	return mac
 end
 
+function get_position()
+	local position = {}
+	uci:foreach("system", "system", function(s) --owm
```

Comment:
I had seen this also and was asking for short, but also think it's not usefull here